gamification-impl/src/main/resources/api-spec.yaml

```diff
@@ -245,6 +302,10 @@ components:
         imageUrl:
           type: string
           nullable: true
+        links:
```

Comment:
Post request should not contain the link
![image](https://user-images.githubusercontent.com/37512627/101378169-0ccd1380-38b3-11eb-93bc-e26e26d65ac0.png)